aroma/plotting.py

```diff
@@ -23,54 +23,40 @@ def classification_plot(in_file, out_dir):
     out_dir : str
         Output directory.
     """
-    # find files
-    myfiles = glob.glob(myinput)
-    LGR.info(f'Found {len(myfiles)} file(s)')
-
-    # load in data from files
-    count = 0
-    for m in myfiles:
-
-        res = []
-
-        tmp = open(m, 'r').read().split('\n')
-
-        for t in tmp[1:-1]:
-            vals = t.split('\t')
-            res.append([vals[1],
-                       float(vals[2]),
-                       float(vals[3]),
-                       float(vals[4]),
-                       float(vals[5])])
-
-        if count == 0:
-            df = pd.DataFrame.from_records(res)
-        else:
-            df2 = pd.DataFrame.from_records(res)
-            df = df.append(df2, ignore_index=True)
-
-        count += 1
+    assert isinstance(in_file, str)
```

Comment:
A nicer way of doing this would be to use the `os` package and an if statement checking that the path exists.

Reply:
This change comes from merging with main I didn't add this line of code

Reply:
I would change it then.

@tsalo any reason why we have an assert here?

Reply:
or maybe i didn't accept the changes when mergin with main. I can't remember now

Reply:
Well, let's see what @tsalo has to say. I guess he added that line.

Reply:
Sorry for the delay on this. I added the assert to make sure that we didn't accept lists of files anymore, I believe.